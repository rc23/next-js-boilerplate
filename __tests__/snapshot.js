import Home from '../pages/index';
import React from 'react';
import renderer from 'react-test-renderer';

it('renders homepage unchanged', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
