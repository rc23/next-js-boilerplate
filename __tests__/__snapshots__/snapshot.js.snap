// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  className="container"
>
  <nav
    className="nav"
  >
    <a
      href="/about"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      About
    </a>
  </nav>
  <main
    className="main"
  >
    <h1
      className="title"
    >
      My Journey!
    </h1>
  </main>
</div>
`;
